Chart: Update maxValueMargin & label.position info (#7509)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChart/1 Configuration/commonAxisSettings/maxValueMargin.md

Controls the empty space between the maximum series points and the axis. Applies only to the axes of the *"continuous"* and *"logarithmic"* type.

---

> The Chart component generates empty space beyond the axis data range to avoid cutting off the largest values. You can control this empty space with **maxValueMargin**. The property is a ratio that is multiplied to the difference between the maximum and minimum values of the axis. The following formula demonstrates how the component uses **maxValueMargin** to calculate axis end values:

Reviewer: ```suggestion
The Chart component adds a space at the end of the axis data range to avoid cutting off the largest values. You can specify the size of this space with **maxValueMargin**, which is a ratio applied to the axis range. The following formula shows how it works:
```

    axisEndValue = maxDataValue + (maxDataValue - minDataValue) * maxValueMargin

> The following expression calculates an axis end value assuming a minimum data value of 1960, maximum data value of 2010, and **maxValueMargin** of 0.1.

Reviewer: ```suggestion
The following expression calculates the axis end value with a minimum data value of 1960, a maximum data value of 2010, and **maxValueMargin** of 0.1:
```
